set.mm

```diff
@@ -815175,6 +815291,359 @@ as an isomorphism in the category of graphs (something like "GraphIsom
+      ZUJKZUUIUUJMZDUKLZNZUUIUUKMZUUQNZUUJUUKMZUUQNZOZOZUFDULLZPZUEUVEPUDUVEPZU
+      UHABDUMLNUVGIUDUEUFBUUQDUVEUVEUNZUUQUNZUOUPAUVFUUHUDUEUVEUVEAUUIUVENZUUJU
+      VENZQZQZUVDUUHUFUVEUVMUUKUVENZQZUVDQZUUICLZUUGNZUUJCLZUUGNZUUKCLZUUGNZOZY
+      JUVQUVSUWAUGZKZYQUVQUVSMZYSNZUVQUWAMZYSNZUVSUWAMZYSNZOZUUHUVOUWCUVDAUVLUV
+      NUWCUVLUVNQZUVJUVKUVNOZAUWCUVJUVKUVNUQACDEURUSNZUVEUUGCVAZUWNUWCRZHCDEUVE
+      UUGUVHUUGUNZUTZUWPUVEUUGCVBZUWQUVEUUGCVCUWTUVJUVRUVKUVTUVNUWBUWTUVJUVRUVE
+      UUGUUICVDVLUWTUVKUVTUVEUUGUUJCVDVLUWTUVNUWBUVEUUGUUKCVDVLVEUPVFVGVHSUVDUV
+      OYJCUULUHZUWDUUMUUOYJUXAKUVCBUULCVIVJUVOUUIUUJUUKUVECUVMCUVEVMZUVNAUXBUVL
+      AUWOUWPUXBHUWSUVEUUGCVKZVFSSAUVJUVKUVNVNUVMUVKUVNAUVJUVKVOSUVMUVNVPVQVRUV
+      PYPUUNUJUVPYJBVSVTZYPUUNKUVPUVEUUGCWAZBUVEWBZBWCNZUXDUVOUXEUVDUVMUXEUVNAU
+      XEUVLAUWOUWPUXEHUWSUVEUUGCWEVFSSSUVPUXFUULUVEWBZUVOUXHUVDUVMUVNUXHUVLUVNU
+      XHRAUVJUVKUVNUXHUUIUUJUUKUVEWFWDWGWHSUVDUXFUXHWIZUVOUUMUUOUXIUVCBUULUVEWJ
+      VJWGWKUVDUXGUVOUUMUUOUXGUVCUUMUXGUULWCNUUIUUJUUKWLBUULWCWMWNVJWGUVEUUGBCW
+      CWOWPYJBWQUPUVOUUMUUOUVCWRWSUVOUVDUWLAUVLUVNUVDUWLRZADWTNZEWTNZUWOUWMUXJR
+      FGHUXKUXLUWOOZUVDUWMUWLUXMUUMUUOUVCUWMUWLRZUXMUVCUXNRUUMUUOUXMUVCCUUPUHZY
+      SNZUUPUVEWBZQZCUUSUHZYSNZUUSUVEWBZQZCUVAUHZYSNZUVAUVEWBZQZOZUXNUXMUURUXRU
+      UTUYBUVBUYFYSCDEUUQUUPUVEUVHUVIYSUNZXAYSCDEUUQUUSUVEUVHUVIUYHXAYSCDEUUQUV
+      AUVEUVHUVIUYHXAXBUWOUXKUYGUXNRZUXLUWOUWPUXBUYIUWSUXCUXBUWMUYGUWLUXBUWMUYG
+      UWLRUXBUWMQZUXRUWGUYBUWIUYFUWKUYJUXPUWGUXQUYJUXPUWGUYJUXOUWFYSUYJUXBUVJUV
+      KUXOUWFKUXBUWMXCZUXBUVJUVKUVNXDZUXBUVJUVKUVNXEZUVEUUIUUJCXGWPTXFXHUYJUXTU
+      WIUYAUYJUXTUWIUYJUXSUWHYSUYJUXBUVJUVNUXSUWHKUYKUYLUXBUVLUVNVOZUVEUUIUUKCX
+      GWPTXFXHUYJUYDUWKUYEUYJUYDUWKUYJUYCUWJYSUYJUXBUVKUVNUYCUWJKUYKUYMUYNUVEUU
+      JUUKCXGWPTXFXHVEVLXIVFXJXKXLXMXIWPVHWHUUFUWEYQUWLOYJUVQYLYMUGZKZYQUVQYLMZ
+      YSNZUVQYMMZYSNZUUDOZOYJUVQUVSYMUGZKZYQUWGUYTUVSYMMZYSNZOZOUAUBUCUVQUVSUWA
+      UUGUUGUUGYKUVQKZYOUYPUUEVUAYQVUGYNUYOYJYKUVQYLYMXQXNVUGYTUYRUUBUYTUUDVUGY
+      RUYQYSYKUVQYLXRTVUGUUAUYSYSYKUVQYMXRTXOXPYLUVSKZUYPVUCVUAVUFYQVUHUYOVUBYJ
+      YLUVSUVQYMXSXNVUHUYRUWGUUDVUEUYTVUHUYQUWFYSYLUVSUVQXTTVUHUUCVUDYSYLUVSYMX
+      RTXPXPYMUWAKZVUCUWEVUFUWLYQVUIVUBUWDYJYMUWAUVQUVSYAXNVUIUYTUWIVUEUWKUWGVU
+      IUYSUWHYSYMUWAUVQXTTVUIVUDUWJYSYMUWAUVSXTTYBXPYGYCYDYEYFUAUBUCYJYSEUUGUWR
+      UYHYHYI $.
+  $}
+
+  ${
+    $d E a b c t y z $.  $d G a b c t y z $.  $d N b c t y z $.
+    $d V a b c t y z $.
+    grtrinbgrex.v $e |- V = ( Vtx ` G ) $.
+    grtrinbgrex.e $e |- E = ( Edg ` G ) $.
+    grtrinbgrex.n $e |- N = ( G NeighbVtx a ) $.
+    $( Conditions for a graph to contain a triangle.  (Contributed by AV,
+       7-Aug-2025.) $)
+    usgrgrtrirex $p |- ( G e. USGraph -> ( E. t t e. ( GrTriangles ` G )
+                                           <-> E. a e. V E. b e. N E. c e. N
+                                           ( b =/= c /\ { b , c } e. E ) ) ) $=
```

Comment:
The labels for the hypotheses `grtrinbgrex` does not match the label for the theorem `usgrgrtrirex`.